CONTRIBUTING.md

```diff
@@ -1,42 +1,67 @@
+
+* Ensure the PR description is precise about the problem and your solution. Just fill out our template. That should cover the most important information.
+
+### Do you intend to add a new feature or change an existing one?
+
+* Suggest your idea via a new GitHub issue. After a confirmation about your idea, you can start writing code. Our maintainers and other project developers can provide useful details about the architecture and show you relevant issues and discussions.
+
+### Do you want to work on translations?
+
+* If you want to improve a translation or add a new translation altogether, we handle those via [POEditor](poeditor).
+
+## Certificate of Origin
+
+By contributing to this project you agree to the [Developer Certificate of
+Origin (DCO)](docs/legal/developer-certificate-of-origin.txt). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. 
+The DCO is a legally binding statement, please [read it carefully](docs/legal/developer-certificate-of-origin.txt).
 
-We use the Developer Certificate of Origin (DCO) as a additional safeguard
-for the HedgeDoc project. This is a well established and widely used
-mechanism to assure contributors have confirmed their right to license
-their contribution under the project's license.
-Please read [docs/legal/developer-certificate-of-origin.txt][dcofile].
 If you can certify it, then just add a line to every git commit message:
 
-```git
-  Signed-off-by: Random J Developer <random@developer.example.org>
+```
+  Signed-off-by: Jane Doe <jane.doe@example.org>
 ```
 
 Use your real name (sorry, no pseudonyms or anonymous contributions).
-If you set your `user.name` and `user.email` git configs, you can sign your
-commit automatically with `git commit -s`. You can also use git [aliases](https://git-scm.com/book/tr/v2/Git-Basics-Git-Aliases)
-like `git config --global alias.ci 'commit -s'`. Now you can commit with
-`git ci` and the commit will be signed.
+
+If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`.
+You can also use git [aliases](https://git-scm.com/book/tr/v2/Git-Basics-Git-Aliases) like `git config --global alias.ci 'commit -s'`.
+Now you can commit with `git ci` and the commit will be signed.
+
+## Submitting a Pull Request
+
+1. Submit an issue describing your proposed change.
+   We will try to respond to your issue as soon as possible.
+2. Fork this repo, develop and test your code changes. Ensure you signed all your commits (see above for details).
+3. Submit a pull request against this repo's `master` branch.
+4. Your branch may be merged once all configured checks pass.
+
+[code-of-conduct]: ./CODE-OF-CONDUCT.md
+[community-forum]: https://community.hedgedoc.org
+[matrix-support]: https://matrix.to/#/#hedgedoc:matrix.org
```

Comment:
`chat.hedgedoc.org` for easier possible future changes of the URL?

Reply:
done